lint_and_test.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -u
+set -ue
```

Comment:
`-e` needs to be there so that the bash script errors out if something (such as lint) fails

Reply:
But now if for example you run this with `everything`, and lint fails, it will not run type checks at all, right? It would be better if we ran all the tests and then errored out if any one of the checks failed, so you would get the most information from that one run, and wouldn't have to run it multiple times.